lua/weapons/gmod_tool/stools/selfchanger.lua

```diff
@@ -1,93 +1,118 @@
+TOOL.Category   = "GUI Panels"
+TOOL.Name       = "Self changer"
+TOOL.Command    = nil
+TOOL.ConfigName = ""
 
-TOOL.Category		= "GUI Panels"
-TOOL.Name			= "Self changer"
-TOOL.Command		= nil
-TOOL.ConfigName		= ""
+TOOL.ClientConVar =
+{
+	["widvariable"] = "",
+	["widvalue"]    = "",
+	["widvaluenum"] = "0"
+}
 
-if ( CLIENT ) then
-    language.Add( "Tool_selfchanger_name", "Developer Tool" )
-    language.Add( "Tool_selfchanger_desc", "changes self....." )
-    language.Add( "Tool_selfchanger_0", "Primary: set self value, Sec: reload draw params" )
-	--language.Add("Tool_selfchanger_widname", "Widget name:")
-	
-	language.Add("Tool_selfchanger_widvariable", "Variable:")
-	language.Add("Tool_selfchanger_widvalue", "Value:")
-	--language.Add("Tool_selfchanger_themename", "Colour scheme:")
-	
-	function umGetNewSet(um)
-		
-		local ent = um:ReadEntity()
-		local var = um:ReadString()
-		local val = um:ReadFloat()
-		Msg("setting "..var.."\n")
-		ent[var] = val
+local function playerNotify(var, val, set, ply)
```

Comment:
I don't like this playerNotify. I'll see if I can find an alternative.

Reply:
Use WireLib.AddNotify

Reply:
Done!